scripts/travis-deploy.sh

```diff
@@ -9,5 +9,6 @@ echo 'Pushing built changes'
 git config --global user.email "cooper@w3.org"
 git config --global user.name "michael-n-cooper"
 git config remote.origin.url "https://${GH_TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git"
-git commit -am "Generated by TRAVIS-CI ${TRAVIS_COMMIT}" -m "${TRAVIS_COMMIT_MESSAGE}"
+git add --all
+git commit -m "Generated by TRAVIS-CI ${TRAVIS_COMMIT}" -m "${TRAVIS_COMMIT_MESSAGE}"
```

Comment:
I am 99% sure we still need the 'a' option here ... the git add command adds untracked files, but to commit all changes in all tracked files, we need '-a' on the commit as well.

Reply:
Think it should be fine without it, but I've added it back anyway